### PR TITLE
Hound-CI test PR

### PR DESCRIPTION
**Pull request by achadwick**
> gui.profiling: partially fix flake8 warnings.
> 
> Just to see what Hound-CI points out when it processes pull requests.
> 

Per https://houndci.com/repos, I need to be submitting pull requests to generate reports. Okay, let's try that. Hound, speak!

### CODE REVIEW

**Comment by achadwick** [comment]
No violations found? Sorry @thoughtbot, but yes there are. Loads of them.

Perhaps "No violations found. Woof!" means "configure me with `fail_on_violations: true`".

```
./gui/symmetry.py:133:9: E731 do not assign a lambda expression, use a def
./gui/symmetry.py:444:80: E501 line too long (82 > 79 characters)
./gui/symmetry.py:475:80: E501 line too long (82 > 79 characters)
./gui/symmetry.py:534:9: F841 local variable 'starts_with_rotate' is assigned to but never used
./gui/symmetry.py:614:5: E303 too many blank lines (2)
./gui/symmetry.py:614:80: E501 line too long (101 > 79 characters)
./gui/symmetry.py:675:9: F841 local variable 'sym_lines' is assigned to but never used
./gui/symmetry.py:735:80: E501 line too long (101 > 79 characters)
./gui/symmetry.py:744:5: C901 'SymmetryOverlay.paint' is too complex (29)
./gui/symmetry.py:754:9: F841 local variable 'axis_rot_symmetry_lines' is assigned to but never used
```

**Comment by achadwick** [comment]
> hound — We've encountered an error while reviewing your code. 

We seem to be getting somewhere now. It would be nice to know ~where~ the style violations are though.

As I understand it, the thing is supposed to comment. No help from their [FAQ](https://intercom.help/hound/frequently-asked-questions/why-isnt-hound-commenting-on-my-pull-request): this PR is trivial, and I believe I have configured it correctly.

File: lib/__init__.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
 
 from __future__ import division, print_function
 
+
+def myfunc(a):
+    """This newly-added function contains several flake8 style violations. In particular, E501, E225, and F821 (twice, or once, depending), E265, and W291. It's only here for testing out Hound-CI."""
+    #You shouldn't write code looking like this.   
+    a+=b
+    return unicode(a)
```
[the remark this answers is not in the document]
Comment (achadwick) [reply lib/__init__.py]: Finally!

Who's a good hound? You are! :dog:

**Comment by achadwick** [comment]
So I guess it only tests newly-added lines. I guess that's reasonable.

**Comment by salbertson** [comment]
> So I guess it only tests newly-added lines. I guess that's reasonable.

@achadwick yep, that's by design. The idea is to comment on current changes only, to maintain focus.

Did you ever get Hound set up the way you wanted?

**Comment by briend** [comment]
@salbertson it drives me nuts, but that's the point :-).  I only wish I could flush all the hound comments to clean up the PR

**Pull request closed without merging**